web/src/main/webapp/xslt/base-layout.xsl

```diff
@@ -54,10 +54,14 @@
 
         <link rel="icon" sizes="16x16 32x32 48x48" type="image/png"
               href="../../images/logos/favicon.png"/>
-        <link href="rss.search?sortBy=changeDate" rel="alternate" type="application/rss+xml"
-              title="{concat($env/system/site/name, ' - ', $env/system/site/organization)}"/>
-        <link href="portal.opensearch" rel="search" type="application/opensearchdescription+xml"
-              title="{concat($env/system/site/name, ' - ', $env/system/site/organization)}"/>
+        <xsl:if test="$env/system/microservicesEnabled = 'true'">
+          <link href="../../api/collections/main/items?f=rss&amp;sortby=-createDate&amp;size=30" rel="alternate" type="application/rss+xml"
+                title="{concat($env/system/site/name, ' - ', $env/system/site/organization)}"/>
+
+          <link href="../../api/collections/main?f=opensearch" rel="search" type="application/opensearchdescription+xml"
```

Comment:
Maybe better to not use relative path in case the XSLT is used in API operation which is not `srv/eng/...`. eg. formatters

```suggestion
          <link href="{/root/gui/url}/api/collections/main/items?f=rss&amp;sortby=-createDate&amp;size=30" rel="alternate" type="application/rss+xml"
                title="{concat($env/system/site/name, ' - ', $env/system/site/organization)}"/>

          <link href="{/root/gui/url}/api/collections/main?f=opensearch" rel="search" type="application/opensearchdescription+xml"
```